poetry/core/masonry/builders/sdist.py

```diff
@@ -302,6 +303,37 @@ def find_nearest_pkg(rel_path):
 
         return pkgdir, sorted(packages), pkg_data
 
+    def find_files_to_add(
+        self, exclude_build=False
+    ):  # type: (bool) -> List[BuildIncludeFile]
+        to_add = super(SdistBuilder, self).find_files_to_add(exclude_build)
+
+        # Include project files
+        logger.debug(" - Adding: pyproject.toml")
+        to_add.add(
+            BuildIncludeFile(
+                path=self._path / Path("pyproject.toml"), source_root=self._path
+            )
+        )
+
+        # If a license file exists, add it
```

Comment:
Shouldn't we adde the following in all builds?

Reply:
Since built distributions are made with source distributions anyway, all of these files will be included. If I add it to all builds, then we will have the `LICENSE` files twice and `README` files twice: once from the `sdist` and once again from the second time `find_files_to_add` will be called from `WheelBuilder._copy_module`.